LayoutTests/inspector/formatting/resources/javascript-tests/optional-chaining.js

```diff
@@ -0,0 +1,3 @@
+foo?.val?.prop;
```

Comment:
can/should we have these be a bit more crazy?  e.g. `foo  ?.  val  ?.  prop  ;` to really make sure formatting is actually happening